runtime/altair/src/lib.rs

```diff
@@ -793,6 +796,7 @@ parameter_types! {
 	pub const PotId: PalletId = common_types::ids::STAKE_POT_PALLET_ID;
 	pub const MaxCandidates: u32 = 1000;
 	pub const MinCandidates: u32 = 5;
+	pub const MaxVoters: u32 = 10 * 1000;
```

Comment:
Seems that we should understand what this value means in case it affect block production indirectly. Is the value coming a live parity chain?

Reply:
Thanks for the comment, I meant to ask the same question about a sane value here. I checked in Parities setup and this was the value I found: https://github.com/paritytech/substrate/blob/e24e9ad5855daef00f64a652ef70ac3848850d56/bin/node/runtime/src/lib.rs#L969

Reply:
Sounds good, usually is more accurate to check on the runtime implementation (actual prod usage) https://github.com/paritytech/polkadot/blob/850d1f5a819de5a9bf52a149ffd8b7d584ba1076/runtime/polkadot/src/lib.rs#L753
And then looking at the meaning of the property: https://github.com/paritytech/substrate/blob/e24e9ad5855daef00f64a652ef70ac3848850d56/frame/elections-phragmen/src/lib.rs#L261-L266
It seems that is limiting the amount of votes taken when doing the onchain election, so if anything is protecting against  higher computation time. 👍